model-optimizer/extensions/middle/FusedBatchNormNonConstant.py

```diff
@@ -40,8 +40,7 @@ def run_before(self):
     def pattern(self):
         return dict(
             nodes=[
-                ('op', dict(kind='op', op=lambda op: op in ['FusedBatchNorm', 'FusedBatchNormV2',
-                                                            'FusedBatchNormV3']))],
+                ('op', dict(kind='op', op=lambda op: op in ['batchNormInference']))],
             edges=[]
         )
 
```

Comment:
We can modify this transformation to be executed during the front phase and no matter that inputs are non constants. All the values will be const-folded automatically later in the pipeline. Note, that methods like `create_node_with_data` should not be used. Instead, `Port`s and `Connection`s should be used.

Reply:
No, these transforms depend on another  middle phase transforms on some models.